ocp_resources/image_stream.py

```diff
@@ -34,27 +32,14 @@ def __init__(
             delete_timeout=delete_timeout,
             **kwargs,
         )
-        self.image_repository = image_repository
-        self.scheduled = scheduled
-        self.tag = tag
+        self.tags = tags
         self.lookup_policy = lookup_policy
 
     def to_dict(self):
         res = super().to_dict()
         if self.yaml_file:
             return res
         res.update(
-            {
-                "spec": {
-                    "lookupPolicy": {"local": self.lookup_policy},
-                    "tags": {
-                        "from": {"kind": "DockerImage", "name": self.image_repository},
-                        "name": self.tag,
-                        "referencePolicy": {"type": "Source"},
-                    },
-                }
-            }
+            {"spec": {"lookupPolicy": {"local": self.lookup_policy}, "tags": self.tags}}
```

Comment:
```"tags": self.tags}```
Is this OK to pass tags: None? Or do we need to do:

```
if tags:
```

Reply:
no, tag should be defined, or else the image stream doesn't work well